### PR TITLE
[doc only] Ganglia removed

### PR DESCRIPTION
**Pull request by jiyoungsin**
-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [[#2230 ](https://github.com/JanusGraph/janusgraph/issues/2230) ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ 2230] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ yes ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ kinda ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ no ] Have you written and/or updated unit tests to verify your changes?
- [ no ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ no ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ no ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ yes ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ yes ] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -161,10 +161,6 @@ after a shiny new JanusGraph header.
     
     `org.apache.tinkerpop.gremlin.server.GremlinServer` => `org.janusgraph.graphdb.server.JanusGraphServer`
 
-##### Drop support for Ganglia metrics
```
Comment (farodin91) [inline docs/changelog.md:164]: You shouldn't remove it from the changelog that we are dropping support for ganglia.

**Review by farodin91 — CHANGES_REQUESTED** [review]
Thank you @jiyoungsin. LGTM beside the one issue. Could squash your commit and include the [doc only] inside the first commit message line, otherwise it will have no impact.

File: docs/changelog.md (hunk 1 above)
Comment (jiyoungsin) [reply docs/changelog.md:164]: So you would like the changelog untouched?

Comment (jiyoungsin) [reply docs/changelog.md:164]: I will create a new pull request with those changes for tomorrow.

Comment (farodin91) [reply docs/changelog.md:164]: You could update this PR.

Comment (mbrukman) [reply docs/changelog.md:164]: @jiyoungsin — once you update your branch locally, just run `git push -f` on your branch to update this PR in-place, no need to create a new PR to address comments.

Comment (jiyoungsin) [reply docs/changelog.md]: @mbrukman thank you for the information! I have pushed a new release. Please let me know if there are any other changes you would like me to complete before merging. I would also like to thank this community for the opportunity to contribute.

Comment (jiyoungsin) [reply docs/changelog.md]: I think i miss understood what i was supposed to do. Was I supposed to just re commit with the. "[doc only]" message?

**Review by farodin91 — APPROVED** [review]
LGTM! 
Before we can merge you to sign the last using `git commit --amend -s --no-edit && git push -f` and I will it squash at the merge. Or you can squash all commits into one commit.

**Comment by jiyoungsin** [comment]
Hey @farodin91 I did "git commit --amend -s --no-edit" && "git push -f" Im not sure if I am supposed to do anything else. .. Can you please clarify if there is anything else I should do?
@mbrukman thank you for the detailed explanation! I did what farodin91 told me to run and could not run your git rebase -i HEAD~3 command. is git rebase similar to git reset?

**Comment by farodin91** [comment]
> Can you please clarify if there is anything else I should do? 

You could squash all your commits into one commit, as described by @mbrukman, which is our normal process. 
It would be also possible that someone merges the PR using the squash command merge it into one commit.

**Comment by jiyoungsin** [comment]
Can you please let me know if I did that correctly?

**Comment by farodin91** [comment]
@jiyoungsin Your commit looks great. You could update the message to be more clear using `git commit --amend`, for example with the following message:

```
ganglia removed [doc only]

Fixes #2230

Signed-off-by: Paul Sin <psin@myseneca.ca>
```

**Comment by mbrukman** [comment]
@jiyoungsin wrote:

> @farodin91 Can you please let me know if I did that correctly?

I am still seeing that you have 2 commits on this branch; squashing them into a single commit would be best. And updating the combined commit message as per @farodin91 would be awesome too.

Thank you for working on this!

**Comment by jiyoungsin** [comment]
Hey guys thanks again for staying with me on this issue, I have just sent a commit with the 3 commit messages squashed into one. Please let me know if I did it correctly :)!

**Comment by mbrukman** [comment]
@jiyoungsin — you still have 2 commits on this branch, did you squash them yet? I think you've just updated the commit message of the last commit.

Take a look at the "Commits" tab on GitHub: https://github.com/JanusGraph/janusgraph/pull/2233/commits and you'll see that there are 2 commits.

Since you have 2 commits, you should run:

```
$ git rebase -i HEAD~2
```

and then the rest of the steps in my earlier long post of squashing and combining the commits together.

When you're done, you should be able to open the "Commits" tab and see only 1 commit listed there.

**Comment by jiyoungsin** [comment]
@mbrukman I thank you for your patience! I just pushed another commit 😰 I think I squashed all of the commits together into one commit and I have edited the commit message.

**Comment by mbrukman** [comment]
@jiyoungsin — I'm not sure what happened, but I think you've pushed the wrong commit, I see this commit in the PR branch now, which is not your commit, but a commit by a bot which updates dependency versions: https://github.com/JanusGraph/janusgraph/pull/2233/commits/5f3afe6be587d39991571b7018fe80d8ad85051c

You might have changed your branch and you may have lost those commits locally (unless you have another pointer to them), but you can find your two commits here:

* (removal) https://github.com/JanusGraph/janusgraph/commit/748a8892b457d80d8df8a1d4967b0271ffa53fda
* (docs update) https://github.com/JanusGraph/janusgraph/commit/ffa06fcbe98dbe590cb286db99026c13c71e725f

To recover them and load them into your local branch, just append `.patch` to each URL:

* (removal) https://github.com/JanusGraph/janusgraph/commit/748a8892b457d80d8df8a1d4967b0271ffa53fda.patch
* (docs update) https://github.com/JanusGraph/janusgraph/commit/ffa06fcbe98dbe590cb286db99026c13c71e725f.patch

That will get you plain-text diffs for each multi-file change. You can save them locally and use [`git apply`](https://git-scm.com/docs/git-apply) to apply them to your local branch to restore your branch to prior state, and then do the squash properly.

**Comment by jiyoungsin** [comment]
I am new to squashing commits so I have created a new pull request with only one Pull Request [here](https://github.com/JanusGraph/janusgraph/compare/master...jiyoungsin:fixes_2230?expand=1). thank you again for your patience and time.

**Comment by mbrukman** [comment]
OK, looks like you've deleted the branch from where you pushed this PR... if you can restore that branch (you should see a button on this page, but we can't do this for you), I can help you fix it, since it looks like you now have a single commit in another branch.

If you don't want to do that, you'd have to open a new PR from your new branch, but if you're open to fixing this PR, I'm happy to help you.

The steps are as follows:

1. restore your branch on this PR via button click, let's call it `[original_branch_name]`
2. `git checkout fixes_2230` since that's the new branch you have created
3. `git branch -D [original_branch_name]` — this deletes the branch, but it's just a pointer, and you'll recreate it shortly
4. `git checkout -b [original_branch_name]` — this recreates the branch that is associated with this PR
5. `git push -f` — this should push `[original_branch_name]` to this PR, updating this PR in-place and getting you exactly the same contents as the `fixes_2230` branch here

and then we should be all set.

**Comment by mbrukman** [comment]
Looks like this PR was replaced by PR #2239, so I'm closing this PR. I've asked @farodin91 to review that PR as well.

**Comment by jiyoungsin** [comment]
Thank you for your help @mbrukman! I hope to get this merged soon :)

**Pull request closed without merging**